ha/core/system_health/bootstrap/health_event_generator.py

```diff
@@ -54,9 +54,8 @@ def create_health_event(self, key, uid, last_modified, status, conf_index, conf_
             rack_id = Conf.get(conf_index, f"server_node{_DELIM}{self._machine_id}{_DELIM}{NODE_MAP_ATTRIBUTES.RACK_ID.value}")
             storageset_id = Conf.get(conf_index, f"server_node{_DELIM}{self._machine_id}{_DELIM}{CONFSTORE_KEY_ATTRIBUTES.STORAGE_SET_ID.value}")
             host_id = Conf.get(conf_index, f"server_node{_DELIM}{self._machine_id}{_DELIM}network{_DELIM}management{_DELIM}public_fqdn")
-
             timestamp = last_modified
-            event_id = timestamp + str(uuid.uuid4().hex)
+            event_id = str(timestamp) + str(uuid.uuid4().hex)
```

Comment:
(timestamp) last_modified is already a string I believe

Reply:
No. When I was debugging this, it was coming as "int"

Reply:
I just printed this:
 ERROR [create_health_event] $$$$$$$$ type of timestamp: <class 'int'>